common/logger/src/lib.rs

```diff
@@ -72,7 +72,10 @@
 
 pub mod prelude {
     pub use crate::{
-        debug, error, event, info, security::SecurityEvent, trace, warn, StructuredLogEntry,
+        debug, error, event, info, sample,
+        sample::{SampleRate, Sampling},
+        security::SecurityEvent,
+        trace, warn, Schema, StructuredLogEntry,
```

Comment:
You're probably going to hit some conflicts with this line being edited in a couple different PRs

Reply:
Yeah, it's expected, but i'll deal with it as it goes.